spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -153,12 +196,11 @@ function onChanOpenTry(
   // only unordered channels allowed
   abortTransactionUnless(order === UNORDERED)
   // assert that version is "ics20-1"
-  abortTransactionUnless(counterpartyVersion === "ics20-1")
+  abortTransactionUnless(counterpartyVersion === "ics20-1" || counterpartyVersion === "ics20-2")
   // allocate an escrow address
   channelEscrowAddresses[channelIdentifier] = newAddress()
-  // return version that this chain will use given the
-  // counterparty version
-  return "ics20-1", nil
+  // return the same version as counterparty version so long as we support it
+  return counterpartyVersion, nil
```

Comment:
Probably I am missing something, but how do we know here if we support the counterparty version?